src/components/RedeCard/RedeCard.stories.js

```diff
@@ -1,12 +1,29 @@
 import React from 'react';
 import { withInfo } from '@storybook/addon-info';
 import RedeCard from './RedeCard';
+import imagem from '../../assets/teacher.png';
 
 export default {
   title: 'RedeCard',
   component: RedeCard,
   decorators: [withInfo],
 };
 
-export const Default = () => <RedeCard description="Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando " title="Testando" />;
-export const NotVisible = () => <RedeCard description="Testando" title="Testando" visible={false} />;
+export const Default = () => (
+  <RedeCard
+    description="Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando Testando "
+    title="Testando"
+  />
```

Comment:
Não seria melhor deixar esse componente com título e descrição vazias ou mais genéricas? Algo como
```suggestion
  <RedeCard
    description="Descrição da mentoria"
    title="Tíitulo da mentoria"
  />
```